guides/common/modules/con_defining-your-content-library.adoc

```diff
@@ -0,0 +1,33 @@
+[id="defining-your-content-library_{context}"]
+= Definining your content library
+
+To ensure that your {ProjectServer} can manage software and provide it to your hosts, you must create repositories and synchronize them.
+
+Red{nbsp}Hat content::
+The Red{nbsp}Hat subscription manifest determines what Red{nbsp}Hat repositories your {ProjectServer} can access.
+Red{nbsp}Hat content is already organized into products.
++
+For example, {RHEL}{nbsp}Server is a product in {Project}.
+The repositories for the {RHEL}{nbsp}Server product consist of different versions, architectures, and add-ons.
+When you enable a Red{nbsp}Hat repository, {Project} automatically creates an associated product.
+
+Other sources of content::
```

Comment:
note for @maximiliankolb create a follow-up PR to reword this for katello and orcharhino builds.

Reply:
Fixed with https://github.com/theforeman/foreman-documentation/pull/3520